@planetarium/account-aws-kms/README.md

```diff
@@ -0,0 +1,40 @@
+@planetarium/account-aws-kms
+============================
+
+An npm package for providing `AwsKmsKeyStore`, an implementation of `KeyStore`
+from *@planetarium/account* that uses AWS KMS as the backend.
+
+Required permissions
+--------------------
+
+Replace `[NUMERIC_ROOT_ACCOUNT_ID]` with your [12-digit root account ID][AWSId]:
+
+~~~~ json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "kms:GetPublicKey",
+        "kms:ScheduleKeyDeletion",
+        "kms:DescribeKey",
+        "kms:ListResourceTags",
+        "kms:Sign",
+        "kms:TagResource"
+      ],
+      "Resource": "arn:aws:kms:*:[NUMERIC_ROOT_ACCOUNT_ID]:key/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "kms:ListKeys",
+        "kms:CreateKey"
+      ],
+      "Resource": "*"
+    }
+  ]
```

Comment:
It can be unimportant, but it will be better to inform users what permissions are needed by each method instead. I feel it seems to say that these permissions are all required and it makes users give the IAM role too many permissions even if it doesn't use all permissions.

Reply:
Amended. PTAL 🙏

Reply:
We've added a detailed table to show required permissions per method:

> | Method                      | Required permissions                | Required for `AwsKmsKeyStoreOptions.scopingTags` |
> |-----------------------------|-------------------------------------|--------------------------------------------------|
> | `AwsKmsKeyStore.list()`     | `kms:ListKeys`                      | `kms:ListResourceTags`                           |
> | `AwsKmsKeyStore.get()`      | `kms:ListKeys`                      | `kms:ListResourceTags`                           |
> | `AwsKmsKeyStore.generate()` | `kms:CreateKey`, `kms:GetPublicKey` | `kms:TagResource`                                |
> | `AwsKmsKeyStore.delete()`   | `kms:ScheduleKeyDeletion`           |                                                  |
> | `AwsKmsAccount.sign()`[^1]  | `kms:Sign`                          |                                                  

[^1]: An `AwsKmsAccount` instance can be obtained from `AwsKmsKeyStore.get()`.